hoot-core/src/main/cpp/hoot/core/algorithms/rubber-sheet/RubberSheet.cpp

```diff
@@ -302,7 +285,7 @@ bool RubberSheet::applyTransform(const OsmMapPtr& map)
 
   _map = map;
 
-  if (!_interpolator2to1)
+  if (!_interpolator2to1 && !_interpolator1to2)
```

Comment:
Because `RubberSheetApplier::apply()` calls `RubberSheet::readTransform1to2()` only `_interpolator1to2` is non-null so the interpolator is never applied.  Either one of the two is required, not just 2to1.